source/macos/AdaptiveCards/AdaptiveCardsTests/Views/ACRTextFieldTests.swift

```diff
@@ -39,7 +39,7 @@ class ACRTextFieldTests: XCTestCase {
     
     func testChangingRenderConfigChangesProperties() {
         let image = NSImage()
-        let inputFieldConfig = InputFieldConfig(height: 40, leftPadding: 20, rightPadding: 20, yPadding: 0, focusRingCornerRadius: 10, borderWidth: 1, wantsClearButton: true, clearButtonImage: image, calendarImage: nil, clockImage: nil, font: .systemFont(ofSize: 10), highlightedColor: .blue, backgroundColor: .yellow, borderColor: .green, activeBorderColor: .black, placeholderTextColor: NSColor.placeholderTextColor, multilineFieldInsets: NSEdgeInsets(top: 5, left: 10, bottom: 0, right: 0))
+        let inputFieldConfig = InputFieldConfig(height: 40, leftPadding: 20, rightPadding: 20, yPadding: 0, focusRingCornerRadius: 10, borderWidth: 1, wantsClearButton: true, clearButtonImage: image, calendarImage: nil, clockImage: nil, font: .systemFont(ofSize: 10), highlightedColor: .blue, backgroundColor: .yellow, borderColor: .green, activeBorderColor: .black, placeholderTextColor: NSColor.placeholderTextColor, multilineFieldInsets: NSEdgeInsets(top: 5, left: 10, bottom: 0, right: 0), errorMessageConfig: ErrorMessageConfig(errorMessageFont: .systemFont(ofSize: 10), errorMessageTextColor: .systemRed, errorBorderColor: .systemRed, errorBackgroundColor: NSColor.systemRed.withAlphaComponent(0.1)))
```

Comment:
Append t this test to see if changingRenderConfigProperties change stuff in textfield